src/components/PhoneButton.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { ReactChild } from 'react';
+import {
+  Image,
+  StyleSheet,
+  Text,
+  TextStyle,
+  TouchableOpacity,
+  ViewStyle,
+} from 'react-native';
+import { theme } from '../styles/colors';
+
+type ButtonProps = {
+  onPress: (event) => void;
+  style?: ViewStyle;
+  labelStyle?: TextStyle;
+  activeOpacity?: number;
+  showShadow?: boolean;
+  children?: ReactChild;
+};
+
+export default function Button(props: ButtonProps) {
+  const shadowStyle = props.showShadow ? styles.buttonShadow : {};
+  const buttonStyle = { ...styles.button, ...props.style, ...shadowStyle };
+  const labelStyle = { ...styles.label, ...props.labelStyle };
+  return (
+    <TouchableOpacity
```

Comment:
Please reuse the existing Button component in this component so this code isn't duplicated. You can pass in styles to customize the styles of Button and it will render children passed into it.

Reply:
I'm not sure how to do this